README.md

```diff
@@ -74,12 +74,14 @@ and copied in Dockerfile. The following files are supported:
        documentation_url. Expects a single key `documentation_url` with a string value
     5. `streams/<stream-name>.patch.json`: to patch a specific stream's document schema
     6. `streams/<stream-name>.pk.json`: to patch a specific stream's primary key, expects an array of strings
+    7. `normalize.json`: to apply data normalization functions to specific fields of documents generated by the connector. Normalizations are provided as a list (JSON array) of objects with keys `pointer` having a value of the pointer to the document field that the normalization will apply to, and `normalization` having a value of the name of the normalization function to apply to the field at that pointer. Normalization function names should provided as `snake_case`; see `airbyte_to_flow/src/interceptors/normalize.rs` for the supported normalization functions.
```

Comment:
What do you think about making this stream-specific? i.e. move this file to `streams/<stream-name>.normalize.json`

I'm just thinking of cases where the key we are normalizing for one stream has a very generic name, e.g. `name`, or `date`, or `timestamp`, and we only want to normalize one of the streams, not all of them.

Reply:
That seems like a really good idea, and will inevitably be necessary. I pushed a change to allow for that.